mesh/mesh.hpp

```diff
@@ -581,6 +582,31 @@ class Mesh
    void Make3D(int nx, int ny, int nz, Element::Type type,
                double sx, double sy, double sz, bool sfc_ordering);
 
+   /** Creates a mesh for the parallelepiped [0,sx]x[0,sy]x[0,sz], divided into
+       nx*ny*nz*24 tetrahedrons. The mesh is generated by taking nx*ny*nz
+       hexahedra and splitting each hexahedron into 24 tetrahedrons.
+       Each face of the hexahedron is split into 4 triangles (face edges are
+       connected to a face-centered point), and the triangles are connected to
+       a hex-centered point.
+   */
```

Comment:
I think we should have brief descriptions for these functions too (admittedly `Make3D` etc. don't have these, but they should)

```suggestion
   /// @brief Creates a mesh for the parallelepiped [0,sx]x[0,sy]x[0,sz], divided into
   /// nx*ny*nz*24 tetrahedrons.
   /// 
   /// The mesh is generated by taking nx*ny*nz
   /// hexahedra and splitting each hexahedron into 24 tetrahedrons.
   /// Each face of the hexahedron is split into 4 triangles (face edges are
   /// connected to a face-centered point), and the triangles are connected to
   /// a hex-centered point.
```

Similarly for all functions below with multiline Doxygen comments.